pkg/logs/client/tcp/destination.go

```diff
@@ -28,7 +28,7 @@ type Destination struct {
 	delimiter           Delimiter
 	connManager         *ConnectionManager
 	destinationsContext *client.DestinationsContext
-	conn                net.Conn
+	connPool            sync.Pool
```

Comment:
In go there is a `zero value` concept which states that if we don't initialise a variable, it will be set to the zero value of its type, see https://golang.org/ref/spec#The_zero_value so if we don't assign.
For example
```
var i int // 0 
var s string // ""
var a *int // nil
...
```